go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/conduitio/conduit-connector-protocol
 go 1.17
 
 require (
+	buf.build/gen/go/conduitio/conduit-connector-protocol/grpc/go v1.3.0-20230307130612-8c27b29b78bf.1
+	buf.build/gen/go/conduitio/conduit-connector-protocol/protocolbuffers/go v1.28.1-20230307130612-8c27b29b78bf.4
```

Comment:
`8c27b29b78bf` mentioned here is a commit from this PR's feature branch. It will need to be updated once we merge the PR.